modules/admin-quota-limits.adoc

```diff
@@ -0,0 +1,303 @@
+====
+
+== Image stream limits
+
+*Supported Resources:*
+
+* `openshift.io/image-tags`
+* `openshift.io/images`
+
+*Resource type name:*
+
+- `openshift.io/ImageStream`
+
+Per image stream, the following must hold true if specified:
+
+.ImageStream
+[cols="3a,8a",options="header"]
+|===
+|Constraint |Behavior
+
+|`Max[openshift.io/image-tags]`
+|`length( uniqueimagetags( imagestream.spec.tags ) )` less than or equal to `Max[openshift.io/image-tags]`
+
+`uniqueimagetags` returns unique references to images of given spec tags.
+
+|`Max[openshift.io/images]`
+|`length( uniqueimages( imagestream.status.tags ) )` less than or equal to `Max[openshift.io/images]`
+
+`uniqueimages` returns unique image names found in status tags. The name is equal to the digest for the image.
+
+|===
+
+== Counting of image references
+
+The `openshift.io/image-tags` resource represents unique xref:../nodes/clusters/nodes-cluster-limit-ranges.adoc#nodes-cluster-limit-stream-limits[stream limits]. Possible references are an `ImageStreamTag`, an `ImageStreamImage`, or a `DockerImage`. Tags can be created by using the `oc tag` and `oc import-image` commands or by using xref:../openshift_images/image-streams-manage.adoc#images-imagestream-use_image-streams-managing[managing images streams]. No distinction is made between internal and external references. However, each unique reference that is tagged in an image stream specification is counted just once. It does not restrict pushes to an internal container image registry in any way, but is useful for tag restriction.
+
+The `openshift.io/images` resource represents unique image names that are recorded in image stream status. It helps to restrict several images that can be pushed to the internal registry. Internal and external references are not distinguished.
+
+
+== PersistentVolumeClaim limits
+
+*Supported Resources:*
+
+* Storage
+
+*Supported Constraints:*
+
+Across all persistent volume claims in a project, the following must hold true:
+
+.Pod
+[cols="3a,8a",options="header"]
+|===
+
+|Constraint |Enforced Behavior
+
+|`Min`
+|Min[<resource>] +<=+ claim.spec.resources.requests[<resource>] (required)
+
+|`Max`
+|claim.spec.resources.requests[<resource>] (required) +<=+ Max[<resource>]
+|===
+
+[[limit-range-obj-def]]
+
+.Limit Range Object Definition
+
+[source,json]
+----
+{
+  "apiVersion": "v1",
+  "kind": "LimitRange",
+  "metadata": {
+    "name": "pvcs" <1>
+  },
+  "spec": {
+    "limits": [{
+        "type": "PersistentVolumeClaim",
+        "min": {
+          "storage": "2Gi" <2>
+        },
+        "max": {
+          "storage": "50Gi" <3>
+        }
+      }
+    ]
+  }
+}
+----
+<1> The name of the limit range object.
+<2> The minimum amount of storage that can be requested in a persistent volume claim.
+<3> The maximum amount of storage that can be requested in a persistent volume claim.
+
+.Additional resources
```

Comment:
This needs to be at the assembly level because of the xrefs. If you had an additional resources section that was just external links, that would be okay - but not xrefs.

Reply:
This is done, thank you!